app/controllers/v0/benefits_documents_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_documents/service'
+
+module V0
+  class BenefitsDocumentsController < ApplicationController
+    before_action { authorize :lighthouse, :access? }
+    Raven.tags_context(team: 'benefits-claim-appeal-status', feature: 'benefits-documents')
+
+    def create
+      params.require :file
+      claim = EVSSClaim.for_user(@current_user).find_by(evss_id: params[:claim_id])
```

Comment:
This table isn't going to be populated anymore after the EVSS deprecation, we should avoid using it. It would probably be worth asking LH what will happen if the claim_id doesn't exist and hope that they return a 404 or something

Reply:
Good idea, I'll look into this.